gateway/api/repositories/jobs.py

```diff
@@ -0,0 +1,31 @@
+"""
+Repository implementation for Job model
+"""
+import logging
+from api.models import Job
+
+logger = logging.getLogger("gateway")
+
+
+class JobsRepository:  # pylint: disable=too-few-public-methods
+    """
+    The main objective of this class is to manage the access to the Job model
+    """
+
+    def get_job_by_id(self, job_id: str) -> Job:
+        """
+        Returns the job for the given id:
+
+        Args:
+            id (str): id of the job
+
+        Returns:
+            Job | None: job with the requested id
+        """
+
+        result_queryset = Job.objects.filter(id=job_id).first()
+
+        if result_queryset is None:
+            logger.warning("Job [%s] was not found", id)
```

Comment:
I think we are using `logger.info` in these cases. `warning` maybe is too much, WDYT?